contrib/oasis.service

```diff
@@ -1,12 +1,16 @@
+# cp oasis.service ~/.config/systemd/user
+# systemctl --user daemon-reload
+# systemctl --user enable oasis
+# systemctl --user start oasis
+
 [Unit]
 Description=Oasis - Friendly Scuttlebutt Web Interface
 Documentation=https://github.com/fraction/oasis
 After=network.target
 
 [Service]
 Type=simple
-User=unicorn
-ExecStart=/usr/bin/node /home/unicorn/.yarn/bin/oasis --host localhost --port 4515
+ExecStart=%s -l -c 'oasis --host localhost --port 4515'
```

Comment:
It took me a minute to understand this, could you double-check that I'm reading this right?

- `%s`: It's important to use the user's shell because they might have a custom `PATH`.
- `-l`: We need to invoke a login shell to make sure we source `.profile` / `.bash_profile` / `.zprofile` / etc
- `-c`: The `ExecStart` command isn't a shell, so unless we want to hardcode the path we need to use a shell.

Reply:
I just tested this locally and noticed that when running with `sysctl --user` it automatically opens my web browser when the service restarts. What do you think about appending `--no-open`?

Reply:
`--no-open` is perfect, will add that shortly